app/models/qa/authorities/local/find_works.rb

```diff
@@ -0,0 +1,23 @@
+module Qa::Authorities
+  class Local::FindWorks < ActiveFedora::Base
+    attr_reader :q
+
+    def initialize(q)
+      @q = q
+    end
+
+    def search(_q, controller)
+      repo = CatalogController.new.repository
+      builder = Sufia::FindWorksSearchBuilder.new(controller)
+      response = repo.search(builder)
+      docs = response.documents
+      results = []
+      docs.each do |doc|
```

Comment:
Instead of doing `each` could you use `map` instead? Then you can also remove line 14 and 20